native_client/client.cc

```diff
@@ -63,7 +63,7 @@ int
 main(int argc, char **argv)
 {
   if (argc < 6 || argc > 7) {
-    printf("Usage: deepspeech MODEL_PATH AUDIO_PATH [-t]\n");
+    printf("Usage: deepspeech MODEL_PATH AUDIO_PATH ALPHABET_PATH LM_PATH TRIE_PATH [-t]\n");
```

Comment:
I'd love that we make the LM non completely mandatory, even though enabled by default :-)